base/rational.jl

```diff
@@ -176,6 +176,8 @@ end
 /(x::Rational, y::Rational) = x//y
 /(x::Rational, z::Complex ) = inv(z/x)
 
+fma(x::Rational, y::Rational, z::Rational) = x*y+z
```

Comment:
IEEE specifies that, if the intermediate value overflows, the final result can be `inf` (or `nan`? can't remember) instead of the correct one. Since we don't track integer (or rational) overflows at all but return wrong results instead, it seems fine to do the same here.